common/src/fixedi128.rs

```diff
@@ -0,0 +1,86 @@
+use fixed_point_math::FixedPoint;
+
+/// Fixed type with inner type of i128 and fixed denominator 10e9
+#[derive(Default, Debug, Clone, Copy, Eq, PartialEq, PartialOrd, Ord)]
+pub struct FixedI128(i128);
+
+impl FixedI128 {
+    pub const DENOMINATOR: i128 = 1_000_000_000;
```

Comment:
I see that we already have denominator in the `rate_math.rs`. Does it make sense to share the same denominator? Or maybe put it as a parameter in the `FixedI128` constructor?

Reply:
rate_math.rs was previous way to work with fixed and should be replaced by FixedI128 and deleted

Reply:
Ok, sounds good to me.

Reply:
In the next PR